acme/agents/jax/r2d2/builder.py

```diff
@@ -247,8 +254,24 @@ def make_actor(
         key='actor_variables',
         update_period=self._config.variable_update_period)
 
+    print('jax devices:', jax.devices())
+    import os
+    print('cuda_visible_devices', os.environ.get('CUDA_VISIBLE_DEVICES'))
+    if force_cpu:
+      print('forcing cpu')
+    actor_backend = 'cpu' if force_cpu else self._config.actor_backend
+    print('actor backend', actor_backend)
+
+    def _obs_map_function(obs):
+      new_observation = OAR(
+        observation=jnp.array(obs.observation),
+        action=jnp.array(obs.action),
+        reward=jnp.array(obs.reward))
+      return new_observation
+
     return actors.GenericActor(
-        policy, random_key, variable_client, adder, backend='cpu')
+        policy, random_key, variable_client, adder, backend=actor_backend,
+        jit=self._config.actor_jit, obs_map_function=_obs_map_function)
```

Comment:
obs_map_function should only be used if we're doing inference server, otherwise its unnecessary copying of data.